pytket/pytket/circuit/display/__init__.py

```diff
@@ -130,35 +185,53 @@ def get_render_options(
         :param _for_js: Whether to convert options to js-compatible format,
             for internal use only.
         """
-        return {
-            (js_key if _for_js else key): self.__getattribute__(key)
-            for key, js_key in self._ALLOWED_RENDER_OPTIONS.items()
-            if full or self.__getattribute__(key) is not None
-        }
+        return self.config.render_options.get_render_options(full, _for_js)
+
+    def save_render_options(self) -> None:
+        """Save the current render options to pytket config."""
+        self.config.update_default_config_file()
 
     def render_circuit_as_html(
-        self, circuit: RenderCircuit, jupyter: bool = False
+        self,
+        circuit: RenderCircuit | List[RenderCircuit],
+        jupyter: bool = False,
+        orient: Orientation | None = None,
     ) -> Optional[str]:
         """
         Render a circuit as HTML for inline display.
 
-        :param circuit: the circuit to render.
+        :param circuit: the circuit(s) to render.
         :param jupyter: set to true to render generated HTML in cell output.
+        :param orient: the direction in which to stack circuits if multiple are present.
         """
-
-        if not isinstance(circuit, Circuit):
-            circuit = Circuit.from_dict(circuit)
+        circuit_dict: dict | List[dict]
+        if isinstance(circuit, list):
+            circuit_dict = [
+                (
+                    circ.to_dict()
+                    if isinstance(circ, Circuit)
+                    else Circuit.from_dict(circ).to_dict()
```

Comment:
Any reason this can't just be `else circ`?

Reply:
So that it throws an error early if the supplied dict isn't a valid circuit